docs/io/asdf-schemas.rst

```diff
@@ -1,105 +0,0 @@
-.. _asdf_schemas:
```

Comment:
This is tricky. Downstream doc that uses intersphinx with this ref will break. @Cadair probably will be affected, if anyone, outside STScI.